webdriver-spec.html

```diff
@@ -8402,6 +8390,80 @@
 </section> <!-- /Take Element Screenshot -->
 </section> <!-- /Screen Capture -->
 
+<section>
+<h2>Get Text Function</h2>
+
+<div class=note>
+<p>One of the major inputs to this specification was the Open
+ Source <a href="http://seleniumhq.org">Selenium</a> WebDriver
+ project. This was in wide-spread use before this specification
+ written, and as such had set user expectations of how the <a>Get
+ Element Text</a> command should work. As such, the function presented
+ here is known to be flawed, but provides the best compatibility with
+ existing users.
+
+<p>The function presented here is
+ the <a href=""><code>bot.dom.getVisibleText</code></a> function
+ compiled with
+ Google's <a href="https://developers.google.com/closure/compiler/">Closure
+ Compiler</a> with advanced optimizations enabled. The code may be
+ recreated by cloning the Selenium project, checking out
+ revision <a href="https://github.com/SeleniumHQ/selenium/commit/c1d39c66115053f71571b9f46d21028bd57ec83f">c1d39c6611</a>
+ and running <code>go //javascript/atoms/fragments:get-text</code>.
```

Comment:
On Windows, `go` on its own is fine. On OS X and Linux, `./` is only needed if the `cwd` isn't on `$PATH`. Adding for clarity.